mars/scheduler/graph.py

```diff
@@ -700,7 +721,7 @@ def create_operand_actors(self, _clean_info=True, _start=True):
         operand_infos = self._operand_infos
 
         op_refs = dict()
-        op_states = dict()
+        meta_op_infos = dict()
```

Comment:
how about `meta_op_infos ` -> `op_meta_infos` ?

Reply:
``meta_op_infos`` means the subset of ``op_info`` to be sent into GraphMetaActor.